src/modules/NetworkController/public/Get-SdnNetworkControllerRestCertificate.ps1

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function Get-SdnNetworkControllerRestCertificate {
+    <#
+    .SYNOPSIS
+        Returns the current Network Controller REST Certificate
+    #>
+
+    try {
+
+        $config = Get-SdnRoleConfiguration -Role 'NetworkController'
+        $confirmFeatures = Confirm-RequiredFeaturesInstalled -Name $config.windowsFeature
+        if (-NOT ($confirmFeatures)) {
+            "The current machine is not a NetworkController, run this on NetworkController or use -NetworkController parameter to specify one" | Trace-Output -Level:Warning
+            return # don't throw exception, since this is a controlled scenario and we do not need stack exception tracing
+        }
+
+        try {
+            $networkController = Get-NetworkController
```

Comment:
need to change this to `Get-SdnNetworkController` which will have fault tolerance if nc is offline